mewarpx/mewarpx/runinfo.py

```diff
@@ -383,9 +366,7 @@ def _save_comp_info(self):
 
     def save(self, diagdir='diags', filename='runinfo.dpkl'):
         """Save run info as a pickle file into the diagnostics directory. The
-        inclusion of injectors forces the use of dill rather than pickle, both
-        owing to customization of Norcross ionization but also apparently
-        scipy.interp1d.
+        inclusion of injectors forces the use of dill rather than pickle.
```

Comment:
I have no idea if this is actually true for WarpX. If you don't either, maybe best to say something like `dill generally handles more diverse objects than pickle, and has historically avoided unpickleable objects in similar situations.`